TASVideos/Pages/Shared/_MovieModule.cshtml

```diff
@@ -55,6 +55,27 @@
 			_ => ""
 		};
 	}
+
+	static string StreamingSiteDisplay(string? onlineWatchingUrl)
+	{
+		if (string.IsNullOrWhiteSpace(onlineWatchingUrl))
+		{
+			return "";
+		}
+
+		var lowered = onlineWatchingUrl.ToLower();
+		if (lowered.Contains("youtube"))
+		{
+			return " on Youtube";
+		}
+
+		if (lowered.Contains("nicovideo"))
+		{
+			return "On Nicovideo";
```

Comment:
To be consistent with youtube, it should read `on` instead of `On`. Also not sure about the difference in leading whitespace.